### ISSUE TITLE
Slowness issue with many tables

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#12332]
From #12324 where the conversation has gotten a little hard to follow (plus it's better tracked as an issue than a pull request).

If I'm remembering correctly, the reporter had issues with speed when there were many tables (I don't recall the number that was stated; either 1,000 or 10,000 sticks in my mind).

The server is running on Debian Wheezy with `select @@lower_case_table_names;` returning "0".

```
git diff libraries/Util.php
diff --git a/libraries/Util.php b/libraries/Util.php
index 5dc34e9..190125b 100644
--- a/libraries/Util.php
+++ b/libraries/Util.php
@@ -4489,7 +4489,7 @@ class Util
if ($lowerCaseTableNames === '0' // issue #10961
|| $lowerCaseTableNames === '2' // issue #11461
) {
- return "COLLATE utf8_bin";
+ return "";
}
return "";
}
```

**Comment by nijel** [comment]
See #10961 and  #11461 for reasoning of this code...

**Comment by nijel** [comment]
This might be also generic INFORMATION_SCHEMA slowness issue, which can be fixed in newer MySQL releases, for example see #12180.
